doc/src/stdlib/collections.md

```diff
@@ -79,6 +79,7 @@ Base.eltype
 Base.indexin
 Base.findin
 Base.unique
+Base.unique!(::AbstractVector)
```

Comment:
I don't think `::AbstractVector` is needed here. If more methods are added later, they would have to be included here too, so better drop it.